distribution/docs/src/main/resources/_contents/_sources/solrcloud-contents.adoc

```diff
@@ -0,0 +1,50 @@
+==== Solr Cloud
```

Comment:
In the `src/main/resources/documentation.adoc` and `src/main/resources/draft-documentation.adoc`, there should be an include statement, such as: `include::{adoc-include}/_contents/_sources/solrcloud-contents.adoc[]`